sparkles/roll_optimize.py

```diff
@@ -309,7 +309,7 @@ def improve_metric(n_stars, P2, n_stars_new, P2_new):
         for roll_interval in roll_intervals:
             roll = roll_interval['roll']
             q_targ_roll = Quat([q_targ.ra, q_targ.dec, roll])
-            q_att_roll = calc_aca_from_targ(q_targ_roll, 0, 0)
+            q_att_roll = self._calc_aca_from_targ(q_targ_roll, 0, 0)
```

Comment:
Looks great.  Though I'm assuming if this wasn't noticed that this could use a quick test.  I could help with that.

Reply:
Sure, thanks!

Reply:
I think this would also be more readable if those were q_att_rolled and q_targ_rolled, but that might just be me.

Reply:
Done.